### PR TITLE
add inngest next.js example

### PR DESCRIPTION
**Pull request by joelhooks**
- [x] The "examples guidelines" are followed from our contributing doc https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md

- [x] Make sure the linting passes by running pnpm build && pnpm lint. See https://github.com/vercel/next.js/blob/canary/contributing/repository/linting.md

### CODE REVIEW

File: examples/inngest/next.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    serverActions: true,
```
Comment (leerob) [inline examples/inngest/next.config.js:4]: This can be removed now! Woohoo!

Comment (joelhooks) [reply examples/inngest/next.config.js]: got this removed. love to see it!

**Review by leerob — DISMISSED** [review]

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: bbb0b713d82c5c6bf26707888262d29d6439731f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: bbb0b713d82c5c6bf26707888262d29d6439731f

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by leerob — APPROVED** [review]

**Review by leerob — DISMISSED** [review]

**Comment by leerob** [comment]
Ah, linting is still failing.

File: docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx [hunk 2]
```diff
@@ -240,7 +240,7 @@ docs](https://developers.google.com/maps/documentation/embed/embedding-map).
 | `allowfullscreen` | Optional | Property to allow certain map parts to go full screen.                                              |
 | `loading`         | Optional | Defaults to lazy. Consider changing if you know your embed will be above the fold.                  |
 | `q`               | Optional | Defines map marker location. _This may be required depending on the map mode_.                      |
-| `center`          | Optional | Defines the center of the map view.                                                                     |
```
Comment (joelhooks) [inline docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx:243]: prettier fix corrected this whitespace at the end of the line, which is unrelated to this PR but I couldn't find any additional prettier-check errors.

File: examples/inngest/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Next.js and Inngest Example
+
+This is an example of how to use [Inngest](https://inngest.com) to easily add durable work flows to your Next.js application. It keeps things simple:
+
+- bare bones examples with a single button UI that triggers an event
+- runs the Inngest dev server locally for immediate feedback
```
Comment (leerob) [inline examples/inngest/README.md:6]:
```suggestion
- Bare bones examples with a single button UI that triggers an event
- Runs the Inngest dev server locally for immediate feedback
```

**Review by leerob — APPROVED** [review]

**Pull request merged**